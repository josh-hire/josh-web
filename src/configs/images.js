export const images = {
  'WEB TESTER': '/assets/logo/mypertamina.png',
};

export default images;
