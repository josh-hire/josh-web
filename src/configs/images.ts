export const images = {
  'WEB TESTER': '/assets/logo/mylogo.png',
};

export default images;
