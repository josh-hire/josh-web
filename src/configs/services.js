export const services = {
  LOGIN: 'user/v1/admins/auth/login',
  GET_CONTACTUS: 'v1/contactus',
};

export default services;
