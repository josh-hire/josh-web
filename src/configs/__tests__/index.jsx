import * as config from '../index';
import * as services from '../services';
import * as images from '../images';

describe('src/configs', ()=>{
  test('test config index',()=>{
    expect(config.IMAGES).toMatchSnapshot();
    expect(config.COOKIE_NAME_TOKEN).toMatchSnapshot();
    expect(config.USER_PROFILE_DATA_STORAGE).toMatchSnapshot();
  });

  test('test config services',()=>{
    expect(services.services).toMatchSnapshot();
  });

  test('test config images',()=>{
    expect(images.images).toMatchSnapshot();
  });

});
