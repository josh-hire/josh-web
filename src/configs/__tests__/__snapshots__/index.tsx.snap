// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/configs test config images 1`] = `
{
  "WEB TESTER": "/assets/logo/mylogo.png",
}
`;

exports[`src/configs test config index 1`] = `
{
  "WEB TESTER": "/assets/logo/mylogo.png",
}
`;

exports[`src/configs test config index 2`] = `"__PT_tester-token"`;

exports[`src/configs test config index 3`] = `"user_profile"`;

exports[`src/configs test config services 1`] = `
{
  "GET_CONTACTUS": "v1/contactus",
  "LOGIN": "user/v1/admins/auth/login",
}
`;
